defaults.json

```diff
@@ -62,18 +62,23 @@
   "admin:county:analyzer": "peliasAdmin",
   "admin:county:field": "parent.county",
   "admin:county:boost": 1,
+  "admin:county:bboxScale": 1.5,
```

Comment:
These variables look out of place because they use `camelCase`.

Could we please change them to use the same convention as the other `multi_word` variable names in this file?

This will help other developers avoid errors since there will only be one naming convention to adhere to and it will (hopefully!) be consistent across the board.

I don't recall the background to this, but that bird has now flown so we're stuck with it how it was done originally 🤷‍♂️

Reply:
fixed